src/bindings/js/node/src/model_wrap.cpp

```diff
@@ -204,3 +205,60 @@ Napi::Value ModelWrap::clone(const Napi::CallbackInfo& info) {
         return info.Env().Undefined();
     }
 }
+
+ov::Output<ov::Node> ModelWrap::input_from_handle(const Napi::Env& env, const Napi::Value& value) {
+    if (ov::js::validate_value<int>(env, value)) {
+        return _model->input(value.As<Napi::Number>().Int32Value());
+    } else if (ov::js::validate_value<Napi::String>(env, value)) {
+        return _model->input(value.As<Napi::String>().Utf8Value());
+    } else if (ov::js::validate_value<Output<ov::Node>>(env, value)) {
```

Comment:
How can we get Output<ov::Node> here?

Reply:
We can pass as key `Model.input` which is of type `Output<ov::Node>` 
I added `test reshape with ports`.

Reply:
Yes, I can see it in the current version.